### DISCUSSION TITLE
Help with Import

### DISCUSSION THREAD

**Discussion by firecerne** [UkrainianBanderasCat/Digital-Logic-Sim2#45] [Q&A] [answered]
Hello !
I am currently remaking the Import Chip system's code because my previous code was very messy .
However, I run into a problem : when I run the game and try importing a chip, it doesn't crash but it returns me an error , and doesn't load the chip :

> NullReferenceException : Object Reference not set to an instance of object

Link to the code : https://pastebin.com/Q2h28Xj2

The error seems to happen on line 104.

Can someone help me resolve this issue ?

**Comment by UkrainianBanderasCat** [comment]
Try to use LIST in usedChipPaths

**Comment by UkrainianBanderasCat** [comment, marked as answer]
```
14 : private List<String> usedChipsPaths = new List<String>();
104 : usedChipsPath.Add(SaveSystem.GetPathToSaveFile(chipName))
```

**Reply by firecerne** [reply]
Thank you ! I'll try it.

**Reply by firecerne** [reply]
It worked !
However, I have another problem with this code : https://pastebin.com/6pyRVX6m
It doesn't give me an error, except this , which is unrelated. It just doesn't work:
![image](https://user-images.githubusercontent.com/105079722/174491065-8cf08eb6-ffdf-4deb-b2c6-f0aed11333fd.png)

It doesn't even copy the files.
Can you help me understanding what is wrong ?
